src/System.Private.ServiceModel/tests/Scenarios/Contract/Service/ServiceKnownTypeTests.cs

```diff
@@ -65,6 +65,20 @@ public static void ServiceKnownType_XmlSerializerFormat_AttrOnType_Test()
         RunTestMethodAndCleanup(factory, serviceProxy, serviceProxy.EchoItems_Xml, new Widget3());
     }
 
+    [Fact]
+    [OuterLoop]
+    public static void ServiceKnownType_XmlSerializerFormat_TwoOperationsShareKnownTypes_Test()
+    {
+        // *** SETUP *** \\
+        ChannelFactory<IServiceKnownTypeTest_AttrOnType_Xml> factory = GetChannelFactory<IServiceKnownTypeTest_AttrOnType_Xml>();
+        IServiceKnownTypeTest_AttrOnType_Xml serviceProxy = factory.CreateChannel();
+
+        // *** EXECUTE *** \\
+        // *** VALIDATE *** \\
+        // *** CLEANUP *** \\
+        RunTestMethodAndCleanup(factory, serviceProxy, serviceProxy.EchoItems_Xml1, new Widget3());
```

Comment:
Thanks for the suggestion. I've added comments in the test. I'll merge the PR upon another signoff.